libs/novui/src/tokens/semanticSpacing.tokens.ts

```diff
@@ -15,4 +15,282 @@ export const SEMANTIC_SPACING_TOKENS = defineSemanticTokens.spacing({
       },
     },
   },
+  paddings: {
+    page: {
+      horizontal: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+      vertical: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+    },
+    panel: {
+      horizontal: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+      vertical: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+    },
+    popover: {
+      horizontal: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+      vertical: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+    },
+    node: {
+      horizontal: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+      vertical: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+    },
+    nav: {
+      menu: {
+        horizontal: {
+          type: 'spacing',
+          value: '{spacing.100}',
+        },
+        vertical: {
+          type: 'spacing',
+          value: '{spacing.100}',
+        },
+      },
+      header: {
+        horizontal: {
+          type: 'spacing',
+          value: '{spacing.50}',
+        },
+        vertical: {
+          type: 'spacing',
+          value: '{spacing.50}',
+        },
+      },
+      futer: {
+        top: {
+          type: 'spacing',
+          value: '{spacing.50}',
+        },
+        bottom: {
+          type: 'spacing',
+          value: '{spacing.100}',
+        },
+      },
+    },
+    WF: {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏I wonder if we should adhere to a semantic property naming convention here? We have a mixture of different cases right now (`camelCase`, `PascalCase`, `Upper-Kebab-Case`, `lower-kebab-case`), the pattern is difficult to assess and follow. My suggestion would be to `camelCase` everything.

Reply:
@rifont I agree, but these are copied directly from Figma. I'm not sure if it's better to try to 

1. standardize them in code but have them differ from Figma
2. Or, keep all of them the same as Figma for consistency with designs (unless they actually break panda)

I opted for #2 now since I'd rather have discussions about how to make our system better when Nik is back. What do you think?